header-featured.php

```diff
@@ -41,7 +41,6 @@
 		<?php endif; ?>
 
 		<link rel='stylesheet' href="<?php echo THEME_STATIC_URL ?>/bootstrap/bootstrap/css/bootstrap-responsive.min.css" type='text/css' media='all' />
```

Comment:
Did the removal of this bootstrap css get missed? Or was this left in on purpose?

Reply:
I originally didn't change the featured header because most of this will be replaced with the changes that are coming up for the feature story template, but it's probably a good idea just to go ahead and remove it for this set of changes for consistency, thanks for catching that!